### ISSUE TITLE
Reschedule failed! Wrong time

### ISSUE THREAD

**Issue by indersinghgill** [uxDaniel/visa_rescheduler#42]
Available dates:
2025-08-18, 2025-08-28, 
Got time successfully! 2025-08-28 09:45
Reschedule Failed!!! 2025-08-28 09:45
Sending notification!

Everything is working fine, apart from rescheduling. I was doing testing with rescheduding and can see it fetches time which is not coming up online and later Reschedule gets failed! Any fix?

**Comment by rafiqlightwala** [comment]
I think despite of the error showing, it actually goes through

**Comment by indersinghgill** [comment]
Yes, you're right it does goes through. I think there is a pop up after rescheduling now and the script is unable to fetch successfully scheduled and reports it as failed.

**Comment by alamoudi10** [comment]
sometimes it goes through, sometimes it does not! I have the same issue. Did anyone figure it out?

**Comment by indersinghgill** [comment]
@alamoudi10 it's because someone else hits reschedule before the script. Lot of people trying these days

**Comment by alamoudi10** [comment]
I see. Well another issue that I have encountered is 
"Got time successfully! XXXX-XX-XX 8:45
break the loop after exception. 

anyone aware of this issue?

**Comment by rafiqlightwala** [comment]
Can anyone confirm if the reschedule request payload is still correct for the website?

**Comment by indersinghgill** [comment]
> Can anyone confirm if the reschedule request payload is still correct for the website?

Yes, it is able to reschedule as last week I got two appointments done. Although it still pushes notification as rescheduled failed.

**Comment by rafiqlightwala** [comment]
I got it last week too - but this week there have been multiple 'rescheduled failed' that have actually failed. I tried to look into the actual request on chrome and they seem slightly different to me than on the script. Can someone try and look into it?

**Comment by indersinghgill** [comment]
@rafiqlightwala I can look but need a account that is looking to reschedule to see if something has been changed.

**Comment by rafiqlightwala** [comment]
I can't share my access - can't you try with your own please :)

**Comment by indersinghgill** [comment]
> 

I got mine rescheduled to June 26th last week only and got visa already :D Helped a friend as well, he got it in September.

**Comment by rafiqlightwala** [comment]
Oh please see if you can check from your friend's account? Since it's in September you can try an earlier date if possible?

**Comment by alamoudi10** [comment]
Dude congrats!! the past week for me it showed barely any appointments. please share your checking time and sleep time?? I have tried to so hard to get it for a friends wedding tomorrow!!! I wish I got mine in June!! please help me out. 

how long has it been since you used the script?
You are lucky!! I never fetched anything in 2023!

**Comment by indersinghgill** [comment]
> Oh please see if you can check from your friend's account? Since it's in September you can try an earlier date if possible?

No sorry, the testing will need to try actually rescheduling a appointment so should be someone who have a appointment booked in 2025 etc so we can try rescheduling and see what is changed. Good luck

**Comment by indersinghgill** [comment]
> please share your checking time

I choose to check in night only, it's the best time. Start at 10pm and keep default time/settings until you get banned.



> how long has it been since you used the script?

It took me one week of proper running daily and my friend two weeks.

**Comment by alamoudi10** [comment]
are you in the ET time zone? 

dude thank you so much! Hopefully it work. I will run it today starting 10pm ET and set the default value. after you get banned did you change the time or did you just let rest and run again? The sleep time is a lot in the default. 
just to confirm whats your default?

**Comment by indersinghgill** [comment]
> are you in the ET time zone?
> 
> dude thank you so much! Hopefully it work. I will run it today starting 10pm ET and set the default value. after you get banned did you change the time or did you just let rest and run again? The sleep time is a lot in the default. just to confirm whats your default?

I'm in Mountain Time:

STEP_TIME = 0.5  # time between steps (interactions with forms): 0.5 seconds
RETRY_TIME = 60*10  # wait time between retries/checks for available dates: 10 minutes
EXCEPTION_TIME = 60*30  # wait time when an exception occurs: 30 minutes
COOLDOWN_TIME = 60*60  # wait time when temporary banned (empty list): 60 minutes

**Comment by alamoudi10** [comment]
Dude are you in Canada or columbia? 😂😂

**Comment by indersinghgill** [comment]
> Dude are you in Canada or columbia? 😂😂

Haha I'm in Canada but it doesn't make much difference, just use for night hours.

**Comment by KaranmaanCa** [comment]
> > Dude are you in Canada or columbia? 😂😂
> 
> Haha I'm in Canada but it doesn't make much difference, just use for night hours.

Hello inder gill, karan this side , i need some info from you as I'm not able to run this properly, please share your email so that i can connect with you .
Thanks
